src/scenes/gameuiscreen.ts

```diff
@@ -559,4 +580,14 @@ export class GameUiScreen extends Phaser.Scene {
         this.dragObj = null;
         this.currentTile.rotation = 0;
     }
+
+    private toggleControls(): void {
+        this.isControlsShowing = !this.isControlsShowing;
+        this.controlsExplained.setVisible(this.isControlsShowing);
+        if (this.isControlsShowing) {
+            this.controlsButton.setTexture("close");
+        } else {
+            this.controlsButton.setTexture("controls");
+        }
```

Comment:
You could do a fancy oneliner: 
`this.controlsButton.setTexture(this.isControlsShowing ? "close" : "controls");`